libtenzir/src/type.cpp

```diff
@@ -742,6 +734,8 @@ const fbs::Type& type::table(enum transparent transparent) const noexcept {
 }
 
 type::operator bool() const noexcept {
+  // TODO: Remove this function?
```

Comment:
Yes please. A type has no "truthiness" .

Reply:
We briefly touched on this yesterday and decided to do the clean ups in follow-up PRs.